cmake/FindSpEC.cmake

```diff
@@ -45,6 +45,29 @@ find_package(MPI COMPONENTS C)
 
 if (SPEC_PACKAGED_EXPORTER_LIB AND SPEC_EXPORTER_FACTORY_OBJECTS AND
     SPEC_EXPORTER_INCLUDE_DIR AND MPI_C_FOUND)
+
+  # Deal with FFTW3
+  #
+  # If it was dynamically linked into SpEC then we need to dynamically link it
+  # into SpECTRE.
+  set(_MACHINE_DEF_FILE
+    ${SPEC_ROOT}/MakefileRules/this_machine.def)
+  set(_FFTW3_LINK_FLAGS "")
+  if (EXISTS ${_MACHINE_DEF_FILE})
+    file(STRINGS ${_MACHINE_DEF_FILE} _MACHINE_DEF)
+    string(REGEX MATCH "FFTW_LIB[^;]+"
+      _FFTW3_DEF "${_MACHINE_DEF}")
+    string(REGEX REPLACE
+      "FFTW_LIB[ ]+=[ ]" ""
```

Comment:
On Wheeler there's no `+=` but this: `FFTW_LIB = /usr/lib64/libfftw3.a`

Reply:
`[ ]+` is searching for 1 or more spaces

Reply:
Ugh right my bad

Reply:
All good! Silly REGEX things :(